Alha javadoc (#7)
hms-networks/sc-flexy-datapoint-lib: README.md

# sc-flexy-datapoint-lib

THE CODE IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND. HMS DOES NOT WARRANT THAT THE FUNCTIONS OF THE CODE WILL MEET YOUR REQUIREMENTS, OR THAT THE OPERATION OF THE CODE WILL BE UNINTERRUPTED OR ERROR-FREE, OR THAT DEFECTS IN IT CAN BE CORRECTED.
---

## [Table of Contents](#table-of-contents)

1. [Description](#description)
2. [Developer Documentation](#developer-documentation)
3. [Dependencies](#dependencies)

---

## [Description](#table-of-contents)

> A library with data point classes for storing information related to and the contents of Ewon historical log data points.

Reviewer: ```suggestion
A library with data point classes for storing the contents of individual Ewon historical log data points.
```
